src/vs/platform/webview/common/webviewContextKeys.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { RawContextKey } from 'vs/platform/contextkey/common/contextkey';
+
+
+// WebView Commands
+export const WEBVIEW_NESTED_IN_LIST_VIEW_FOCUSED = new RawContextKey<boolean>('webviewNestedInListViewFocused', false);
```

Comment:
Not sure whether this should stay here or under `platform/webview` or under `platform/list/browser`

Reply:
I wonder if we could have a generic "webviewFocused" or "focusInWebview", since we always want ctrl+a to be delegated to webview if it has focus, no matter if it's inside a list view or not.

Reply:
Done